docs/demo/gaiden-css/generic/headings/demo.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="maximum-scale=1,width=device-width,initial-scale=1,user-scalable=0">
+ <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    nav {
+      padding: 20px;
+    }
+
+  </style>
+  <title>Headings</title>
+</head>
+
+<body>
+    <h1 class="" arial-level="1" role="heading">
+      H1 Title Gaiden
+      <small>Sub Title Gaiden</small>
```

Comment:
Looking your example I don't see an association with implementation of classes and small.

To clarify, h2-6 exists to be a sublevel of heading, not to define visually a subtitle. Our category page have a little "caption" on H1, but this information is not a subtitle of section.